For some files on cross, C++ requires the cmath header, when only the C
math.h header is included. Include cmath where nessecary.

--- third_party/angle/src/compiler/translator/InfoSink.h	2020-09-26 11:53:15.195689884 +1000
+++ third_party/angle/src/compiler/translator/InfoSink.h	2020-09-26 12:00:55.341995386 +1000
@@ -7,7 +7,12 @@
 #ifndef COMPILER_TRANSLATOR_INFOSINK_H_
 #define COMPILER_TRANSLATOR_INFOSINK_H_
 
-#include <math.h>
+#ifdef __cplusplus
+# include <cmath>
+using std::signbit;
+#else
+# include <math.h>
+#endif
 #include <stdlib.h>
 #include "compiler/translator/Common.h"
 #include "compiler/translator/Severity.h"
--- third_party/skia/src/sksl/SkSLString.cpp	2020-09-26 17:43:56.042840919 +1000
+++ third_party/skia/src/sksl/SkSLString.cpp	2020-09-26 17:44:25.480961135 +1000
@@ -15,6 +15,13 @@
 #include <sstream>
 #include <string>
 
+#ifdef __cplusplus
+# include <cmath>
+using std::signbit;
+#else
+# include <math.h>
+#endif
+
 namespace SkSL {
 
 String String::printf(const char* fmt, ...) {
